Update README.md (#70)
Imageomics/kabr-tools: README.md

In order to automatically label the animal videos with behavior, we must first create *mini-scenes* of each individual animal captured in the frame, illustrated below.

See the Wiki [CVAT User Guide](https://github.com/Imageomics/kabr-tools/wiki/CVAT-User-Guide) and [Data Management Tips](https://github.com/Imageomics/kabr-tools/wiki/Data-Management-Tips) for detailed instructions and recommendations.

![](images/im_mini-scenes.jpg)
**Figure 3:** A mini-scene is a sub-image cropped from the drone video footage centered on and surrounding a single animal. Mini-scenes simulate the camera as well-aligned with each animal in

> the frame, compensating for the drone's movement and ignoring everything in the large field of
> view but the animal’s immediate surroundings. The KABR dataset consists of mini-scenes and their

Reviewer: ```suggestion
the frame, compensating for the drone's movement by focusing on just the animal and its immediate surroundings. The KABR dataset consists of mini-scenes and their
```
Would it be more appropriate to describe it as "and focusing..." or "by focusing..." I think that focus is at least part of how it compensates for the drone movement?

Author: yes, i like 'focusing' a lot better here
